examples/WritingSALScript.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<p>setup <em>the</em> Python logging....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>